src/commons/achievement/AchievementView.tsx

```diff
@@ -84,7 +100,9 @@ function AchievementView({ focusUuid, courseRegId }: AchievementViewProps) {
         selectedAssessment &&
         selectedAssessmentConfig &&
         selectedAssessmentConfig.isManuallyGraded && (
-          <AchievementCommentCard courseId={courseId} assessment={selectedAssessment} />
+          // TODO: showToQuestion is currently used to disable the goto question button for admins,
+          // as it has not been integrated with the grading view yet
+          <AchievementCommentCard assessment={selectedAssessment} showToQuestion={!isAdminView} />
```

Comment:
Any updates on this?

Reply:
it'll be a non-trivial addition as we will need to find a way to fetch the submission id from the backend, based on the question and mission ids (iirc). I haven't had the time to work on this, perhaps the team next sem can try to figure this out?

Reply:
Ok could you file an issue for this? Thanks

Reply:
https://github.com/source-academy/frontend/issues/2303